src/main/java/firebase/sdk/dashboard/api/ReleaseResource.java

```diff
@@ -0,0 +1,122 @@
+package firebase.sdk.dashboard.api;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Date;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import firebase.sdk.dashboard.dao.PlatformReleaseDao;
+import firebase.sdk.dashboard.dao.PlatformReleaseDaoDatastore;
+import firebase.sdk.dashboard.dao.SDKDao;
+import firebase.sdk.dashboard.data.SDK;
+import firebase.sdk.dashboard.data.SDKRelease;
+import firebase.sdk.dashboard.data.Release;
+
+/**
+ * Root resource (exposed at "platforms/{platform}/releases" path)
+ */
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class ReleaseResource {
+
+  private static final PlatformReleaseDao PRDAO = new PlatformReleaseDaoDatastore();
+//  private static final SDKDao SDAO = new SDKDao();
+  private String platform;
+  private ArrayList<String> sdks = new ArrayList<>();
+
+  public ReleaseResource(String platform) {
+    this.platform = platform;
+    // Dummy Code for API
+    sdks.add("firebase-common");
+    sdks.add("firebase-common-ktx");
+    sdks.add("firebase-ml");
+    sdks.add("firebase-database");
+    sdks.add("firebase-auth");
+    sdks.add("firebase-components");
+  }
+
+  /**
+   * Method handling HTTP GET requests. The returned object will be sent
+   * to the client as "text/plain" media type.
+   *
+   * @return String that will be returned as a text/plain response.
+   */
+  @GET
```

Comment:
Path?

Reply:
This endpoint doesn't require an explicit path annotation because that is handled in the platform resource file. However I am adding the paths to the java docs of each endpoint method.